server/migrations/2021-03-07-173414_create_rooms/up.sql

```diff
@@ -0,0 +1,17 @@
+CREATE EXTENSION IF NOT EXISTS "uuid-ossp";
+
+CREATE TABLE "rooms"
+(
+    id UUID PRIMARY KEY DEFAULT uuid_generate_v4(),
+    name VARCHAR
+);
+
+CREATE TABLE "rooms_users"
+(
+    user_id UUID,
+    room_id UUID,
+    status VARCHAR,
```

Comment:
What is the status used for?

Reply:
Inspired by matrix. Status can be used for something like "pending", "banned", "accepted" and so on to describe the relationship between the user and the room. Could also be done with another foreign key but I did not add another table since it is currently not used.

Reply:
Alright